gen.go

```diff
@@ -308,18 +308,15 @@ func fetchISO() (map[string]*currencyInfo, error) {
 
 	currencies := make(map[string]*currencyInfo, 170)
 	for _, entry := range aux.Table[0].Entry {
-		if entry.Code == "" || entry.Name.IsFund {
+		if entry.Code == "" || entry.Number == "" || entry.Digits == "N.A." {
 			continue
 		}
-		if entry.Code == "XUA" || entry.Code == "XSU" || entry.Code == "XDR" {
-			continue
-		}
-		if strings.HasPrefix(entry.Country, "ZZ") {
-			// Special currency (Gold, Platinum, etc).
-			continue
+
+		digits, err := strconv.Atoi(entry.Digits)
```

Comment:
This change doesn't actually improve the logic because the digits are never used, they are replaced via replaceDigits() in the next call. So it's fine to keep using parseDigits(), possibly with a new comment explaining the situation.

Reply:
Actually, as I investigated, digits from ISO could be used in case they're not available in CLDR data. Added comment on that. 

Added log to check what currencies don't have digits in CLDR (in replaceDigits func):
```
	for currencyCode := range currencies {
		fractions, ok := aux.Supplemental.CurrencyData.Fractions[currencyCode]
		if ok {
			currencies[currencyCode].digits = parseDigits(fractions["_digits"], 2)
		} else {
			fmt.Println("Warning: no digits for", currencyCode)
		}
	}
```

and it was the result:
```
Warning: no digits for MDL
Warning: no digits for RUB
Warning: no digits for CHE
Warning: no digits for KYD
Warning: no digits for KZT
Warning: no digits for MRU
Warning: no digits for NPR
Warning: no digits for NIO
Warning: no digits for SAR
Warning: no digits for BWP
Warning: no digits for ANG
Warning: no digits for GMD
Warning: no digits for INR
Warning: no digits for EGP
Warning: no digits for PLN
Warning: no digits for BOB
Warning: no digits for UAH
Warning: no digits for VED
Warning: no digits for AOA
Warning: no digits for BMD
Warning: no digits for MXN
Warning: no digits for PGK
Warning: no digits for SHP
Warning: no digits for LKR
Warning: no digits for TTD
Warning: no digits for BND
Warning: no digits for SZL
Warning: no digits for SDG
Warning: no digits for EUR
Warning: no digits for BTN
Warning: no digits for CNY
Warning: no digits for LRD
Warning: no digits for BZD
Warning: no digits for MXV
Warning: no digits for SBD
Warning: no digits for ARS
Warning: no digits for TJS
Warning: no digits for MWK
Warning: no digits for SGD
Warning: no digits for LSL
Warning: no digits for BDT
Warning: no digits for SVC
Warning: no digits for ZAR
Warning: no digits for STN
Warning: no digits for TOP
Warning: no digits for CUC
Warning: no digits for GBP
Warning: no digits for HNL
Warning: no digits for MAD
Warning: no digits for DZD
Warning: no digits for BOV
Warning: no digits for CHW
Warning: no digits for SCR
Warning: no digits for SSP
Warning: no digits for NAD
Warning: no digits for HKD
Warning: no digits for UYU
Warning: no digits for USD
Warning: no digits for CUP
Warning: no digits for MZN
Warning: no digits for PAB
Warning: no digits for USN
Warning: no digits for BAM
Warning: no digits for BGN
Warning: no digits for MKD
Warning: no digits for WST
Warning: no digits for CVE
Warning: no digits for FKP
Warning: no digits for QAR
Warning: no digits for SRD
Warning: no digits for DOP
Warning: no digits for CDF
Warning: no digits for FJD
Warning: no digits for GIP
Warning: no digits for NGN
Warning: no digits for THB
Warning: no digits for XCD
Warning: no digits for AUD
Warning: no digits for ZMW
Warning: no digits for GHS
Warning: no digits for PEN
Warning: no digits for TMT
Warning: no digits for AED
Warning: no digits for BBD
Warning: no digits for HRK
Warning: no digits for GTQ
Warning: no digits for HTG
Warning: no digits for MVR
Warning: no digits for JMD
Warning: no digits for NZD
Warning: no digits for GEL
Warning: no digits for MYR
Warning: no digits for TRY
Warning: no digits for AZN
Warning: no digits for COU
Warning: no digits for VES
Warning: no digits for RON
Warning: no digits for BRL
Warning: no digits for ILS
Warning: no digits for KGS
Warning: no digits for ZWL
Warning: no digits for BSD
Warning: no digits for ERN
Warning: no digits for MOP
Warning: no digits for PHP
Warning: no digits for AWG
Warning: no digits for KHR
Warning: no digits for KES
Warning: no digits for ETB
```